backend/src/test/java/de/teamprojectfinance/backend/PostingServiceTest.java

```diff
@@ -38,6 +41,19 @@ void whenPostindAdded_ThenReturnPosting() {
 
 
  }
-
-
+@Test
+    void whenListNotEmpty_ThenReturnList(){
+     // Given
+    PostingModel givenTestModelRepo = new PostingModel("2B", "test2", 23, PostingCategory.INCOME);
+    PostingModel givenTestModel1 = new PostingModel("3C", "test3", 23, PostingCategory.EXPENSE);
+    List<PostingModel> givenTestPostigModel = new ArrayList<>(List.of(givenTestModel1,givenTestModelRepo));
```

Comment:
your Posting Models have a weird naming convention, stick to more consistent variable naming. Also you don't have to write redundant variables here and just initialize the List.of with new Instantiations.
Also you don't write new ArrayList<> here, since you dont want to create an ArrayList which has on its first Index another ArrayList. What you wrote here is an array in an array.
Better do it that way:

```suggestion
    List<PostingModel> givenTestPostigModel = List.of(
            new PostingModel("2B", "test2", 23, PostingCategory.INCOME),
            new PostingModel("3C", "test3", 23, PostingCategory.EXPENSE)
    );
```

Reply:
Thanks for your detailed and good suggestions. But in one thing you are wrong. There is no further ArrayList in the Arraylist. Only the initialization and the filling with List.of().